pennylane/fourier/__init__.py

```diff
@@ -11,6 +11,21 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-r"""Tools to compute the Fourier series representation of quantum circuits."""
+r"""Tools to compute and visualise the Fourier series representation of quantum circuits."""
```

Comment:
I wonder if we should delete this so it does not show in the docs? It is kind of confusing when we just introduced the package...